testsuite/testenv.sh

```diff
@@ -148,6 +148,15 @@ synth_tb()
   clean
 }
 
+# Check if a C compiler is installed on this system
+c_compiler_is_available ()
+{
+  if [ -z $CC ]; then
+    CC="gcc"
+  fi
+  which $CC
+}
+
 # Check if a feature is present
 ghdl_has_feature ()
```

Comment:
Might be worth calling `c_compiler_is_available` from inside `ghdl_has_feature`. That will avoid most of the modifications to individual tests (except bug097 and issue1326).